src/test/osd/osd-scrub-repair.sh

```diff
@@ -908,7 +908,12 @@ function TEST_corrupt_scrub_replicated() {
 EOF
 
     jq "$jqfilter" $dir/json | python -c "$sortkeys" | sed -e "$sedfilter" > $dir/csjson
-    diff -y $termwidth $dir/checkcsjson $dir/csjson || test $getjson = "yes" || return 1
+    if [ `uname` != FreeBSD ]; then
```

Comment:
@tchaikov @dzafman 
Eh, Yes this ended up in the wrong PR.
But given the dislike of IGNORE_UNUSED(), I will close this PR and submit a different one